specs/configuration_files_specification.md

```diff
@@ -73,6 +73,17 @@ The configuration in `bar.conf` has the lowest priority,
 and is overridden by `a.conf` and `b.conf`.
 `b.conf` has the highest priority.
 
+Here's another, more robust example. This is the order in which the files are parsed.
```

Comment:
I don't like this introductory sentence. This is essentially the same example, but with more details. Can you combine the two paragraphs into a single description? Maybe something like: "Considering the following files present in the file system … (and then most of the text of the first paragraph)".